docs/source/api_ref_training.rst

```diff
@@ -76,6 +76,20 @@ Utilities to reduce memory consumption during training.
 
 .. _metric_logging_label:
 
+Schedulers
+-----------------
+
+Utilities to control lr during the training process.
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    get_cosine_schedule_with_warmup
+
+.. lr_scheduler_label:
```

Comment:
this should swap with metric_logging_label above, and should have an underscore before: `.. _lr_scheduler_label:`. otherwise it gets interpreted as a comment (sphinx, I know)

Reply:
Fixed